hedyorg/hedy
# [UI] Add public profile link to account dropdown

**Description**

Removes open profile link button in profile settings and adds a new button in profile dropdown menu which links to open profile

**Fixes #4445**

**How to test**

* Login as teacher1 or any other teacher account
* Click on the account in the top right side of the site and see the dropdown
* A new menu item should be located under the account setting and before the exit button
* In account setting the green button should disappear 

**Checklist**
Done? Check if you have it all in place using this list: (mark with x if done)

- [ x] Contains one of the PR categories in the name
- [ x] Describes changes in the format above
- [ x] Links to an existing issue or discussion
- [ x] Has a "How to test" section

If you're unsure about any of these, don't hesitate to ask. We're here to help!
